src/main/java/htsjdk/samtools/util/SortingCollection.java

```diff
@@ -320,7 +320,7 @@ public void cleanup() {
                                         codec,
                                         comparator,
                                         maxRecordsInRAM,
-                                        tmpDirs.toArray(new File[tmpDirs.size()]));
+                                        tmpDirs.stream().map(File::toPath).toArray(Path[]::new));
```

Comment:
untested

Reply:
I also deprecated this method. I do not think that it is worthy to add a test for it...